src/ZEO/runzeo.py

```diff
@@ -193,11 +191,12 @@ def can_connect(self, family, address):
             return 1
 
     def clear_socket(self):
-        if isinstance(self.options.address, type("")):
+        if isinstance(self.options.address, six.string_types):
             try:
                 os.unlink(self.options.address)
             except os.error:
                 pass
+            return True
```

Comment:
Why are we returning `True` here?  And I don't see test coverage below for the branch where the address is *not* a string.